vimeo-networking/src/main/java/com/vimeo/networking/Configuration.java

```diff
@@ -55,6 +55,8 @@
     public String clientSecret;
     public String scope;
 
+    public String accessToken;
```

Comment:
I think in order to realize the full pattern of the Builder, we need to make these fields private and have getters for them - this way the only true way to change them is through the Builder. I suggest doing that here for accessToken and creating a ticket for the other fields.